### PR TITLE
Add script for server side help

### PR DESCRIPTION
**Pull request by dveeden**
### What is changed, added or deleted? (Required)

Add script to create help data for https://github.com/pingcap/tidb/pull/25485

This takes the statement documentation and creates insert statements from that for `mysql.help_topic`

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### CODE REVIEW

**Review by morgo — APPROVED** [review]

**Review by xhebox — APPROVED** [review]
I thought go embed need a real existed file. But looks like we did not generate any file yet.. Maybe add a example file for `help` command itself?

**Comment by dveeden** [comment]
> I thought go embed need a real existed file. But looks like we did not generate any file yet.. Maybe add a example file for `help` statement itself?

Yes, that's a bit of an issue. The files `go embed` uses are created by `scripts/server_help.sh` but are not added as part of the commit as this is excluded in `.gitignore`. We could change that and include it, just like `parser.go` in pingcap/parser.

There is another option: embed the `sql-statements/sql-statement-*md` files and add some golang dependencies for the markdown to text conversion.

My choice would be the first option, were we use pandoc to convert from markdown to plain text.

**Comment by xhebox** [comment]
> My choice would be the first option

I like how it is done now :)

It is just, seems we can not really test the result of this PR in TiDB... But I gave LGTM anyway. There is nothing wrong from my point of view.

File: .circleci/config.yml [hunk 1]
```diff
@@ -112,6 +112,26 @@ jobs:
       - store_artifacts:
           path: /tmp/artifacts
 
+  server_help:
```
Comment (TomShawn) [inline .circleci/config.yml:115]: Do we really need to add this to CircleCI check that runs for every `docs` commit?

**Pull request closed without merging**